ADR/xxxx-kite-poc-architecture-and-components.md

```diff
@@ -12,6 +12,121 @@ The [Konflux Issue Tracking Engine](https://github.com/konflux-ci/kite) (KITE) i
+
+**Value**:
+- Time saved (e.g. 5-30 minutes daily)
+- Immediate awareness of all blocking issues
+
+
+**2. Cross-Component Failure Correlation**
+
+**Scenario**: A developer notices that his application is failing but doesn't realize it's related to a shared library issue affecting multiple teams.
+
+**Current Problem**: Multiple teams independently debug the same root cause, wasting collective hours.
+
+**With Issues Dashboard**:
+- Single issue: "Shared library v2.1 causing build failures"
+- **Scope**: Components affected by the same issue can fall under the same [scope](https://github.com/konflux-ci/kite/blob/main/packages/backend/docs/Webhooks.md#issue-grouping-with-scope-objects).
+- **Grouping**: By grouping the issues, multiple teams are alerted of the same error, preventing separate debugging efforts.
+**Auto-resolution**: When the library is fixed, all related issues can be [resolved automatically](https://github.com/konflux-ci/kite/blob/main/packages/backend/docs/Webhooks.md#automatic-issue-resolution-via-custom-webhooks) because of the shared scope.
+
+**3. User Support Rotation**
+
+**Scenario**: I am an associate on a Konflux Support rotation. A user complains that their application won't build and they don't know why
+
+**Current Problem**:
+- Depending on the supporting associates skill set, they may or may not have the knowledge of how to investigate such issues.
+  - Querying a Konflux cluster and checking the logs for a failing pipelinerun can require multiple skills that are out of reach for a supporting associate.
+- The user might have to navigate through multiple views in order to get a clear picture of what could be the cause.
+- A power user capable of investigating logs for a CR on a Konflux cluster still needs to figure out where to check.
+
+**With Issues Dashboard**:
+- A non-technical associate can point to the Issues Dashboard and point them towards three issues related to failed builds.
+  - Each issue has logs to those builds, allowing for deeper investigation.
+- A power-user associate can use the [KITE CLI tool](https://github.com/konflux-ci/kite/tree/main/packages/cli) to quickly check if any existing issue records exist in the namespace(s) the user has access to.
+  - The CLI tool reports three build failures in namespace `team-gamma`, with logs for each failure.
+
+**Value**:
+- Non-technical support associates can leverage the issues dashboard to quickly show them what is happening where and why.
+- Power-user support associates can quickly find the information they need using the CLI tool, getting them closer to where deeper analysis is needed.
+- Regular users also have access to these tools, potentially reducing the amount of support tickets filed.
+
+**4. Security Vulnerability Response**
+
+**Scenario**: A critical CVE is discovered in a widely-used dependency (e.g. a container base image, Python package, or RPM package) that affects multiple components across different teams.
+
+**Current Problem**:
```

Comment:
IMO, this should be solved by running pipelines that the user need to have access to in the user's namespace and include the CR which is responsible for triggering the pipelines in it as well. This will give the user visabilllity on the the plrs and the starus of the cr
 Another approach is to includec additional information in the status field of the component.